src/dune_rules/opam_create.ml

```diff
@@ -228,6 +228,7 @@ let opam_fields project (package : Package.t) =
   in
   let list_fields =
     [ "maintainer", Package_info.maintainers info
+    ; "x-maintenance-intent", Package_info.maintenance_intent info
```

Comment:
Shouldn't this be inserted only if the field is not `None`?

Reply:
Yes if I understand correctly there's a `filter_map` right after that removes any field with a useless `None|Some []` value :) (so if `maintenance_field` is not specified, it doesn't appear in the produced opam file). Did you have something else in mind?

Reply:
No, just wanted to be sure :smile: